### PR TITLE
[vscode][5/n] Setup Env Variables: Read the text from the env template and append to existing file

### PR DESCRIPTION
**Pull request by rossdanlm**
[vscode][5/n] Setup Env Variables: Read the text from the env template and append to existing file

TSIA, and making the function async now that we call an await statement

## Test Plan

https://github.com/lastmile-ai/aiconfig/assets/151060367/6679e13d-6d58-4ec4-a237-360cb273e59c

---
Stack created with [Sapling](https://sapling-scm.com). Best reviewed with [ReviewStack](https://reviewstack.dev/lastmile-ai/aiconfig/pull/1285).
* __->__ #1285
* #1284
* #1283
* #1280

### CODE REVIEW

File: vscode-extension/src/extension.ts [hunk 1]
```diff
@@ -757,18 +757,29 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
-    vscode.window.showInformationMessage(
-      "Env file already exists, will implement next PR"
-    );
+    const helperText = (
+      await vscode.workspace.fs.readFile(envTemplatePath)
+    ).toString();
+
+    // TODO: Check if we already appended the template text to existing .env
+    // file before. If we did, don't do it again
+    fs.appendFile(envPath, "\n\n" + helperText, function (err) {
```
Comment (saqadri) [inline vscode-extension/src/extension.ts:773]: This is fine for now, but if a file already exists perhaps we should ask the user to specify a different file or ask them if they want to overwrite or append. Can be a future improvement.

File: vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -764,17 +764,13 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
-    var helperText: string = "\ntest, will change next PR";
-    // fs.readFile(envTemplatePath.fsPath, function read(err, data) {
-    //   if (err) {
-    //     throw err;
-    //   }
-    //   helperText = "\n" + data.toString();
-    // });
+    const helperText = (
+      await vscode.workspace.fs.readFile(envTemplatePath)
+    ).toString();
```
Comment (Ankush-lastmile) [inline vscode-extension/src/extension.ts:769]: nice! using const instead of let

File: vscode-extension/src/extension.ts [hunk 3]
```diff
@@ -764,17 +764,13 @@ async function setupEnvironmentVariables(context: vscode.ExtensionContext) {
...
 
     // TODO: Check if we already appended the template text to existing .env
     // file before. If we did, don't do it again
-    fs.appendFile(envPath, helperText, function (err) {
+    fs.appendFile(envPath, "\n\n" + helperText, function (err) {
```
Comment (Ankush-lastmile) [inline vscode-extension/src/extension.ts:773]: nit: it might make more sense to define a constant for this

**Review by saqadri — APPROVED** [review]

**Review by Ankush-lastmile — APPROVED** [review]
lgtm, approving to unblock

File: vscode-extension/src/extension.ts (hunk 1 above)
Comment (rossdanlm) [reply vscode-extension/src/extension.ts:773]: I'm going to push back saying this is P1 and focus more on content for tonight. If I have time before midnight I'll do it

**Pull request merged**